tests/unit/test_py_vs_cpp.py

```diff
@@ -78,21 +78,25 @@ def compare_context(
     py_context: ParserRuleContext, cpp_context: ParserRuleContext
 ) -> None:
     assert type(py_context) == type(cpp_context)
-    assert len(py_context.children) == len(cpp_context.children)
+    py_context_children = list(py_context.getChildren())
```

Comment:
Was there a clear cut problem with using the `children` attribute here and in the following lines or was this syntactic preference?